trailblazer/store/crud/delete.py

```diff
@@ -0,0 +1,12 @@
+from trailblazer.store.base import BaseHandler_2
+from trailblazer.store.models import Analysis
+
+
+class DeleteHandler(BaseHandler_2):
+    """Class for deleting items in the database."""
+
+    def delete_analysis_jobs(self, analysis: Analysis) -> None:
+        """Delete jobs in the Jobs table."""
+        for job in analysis.failed_jobs:
+            job.delete()
```

Comment:
I think `object.delete()` is some alchy magic. This works as well and is how we do it now in `cg`
```suggestion
           self.session.delete( job)
```

Reply:
No, this will not work out of the box and will have to wait until we have refactored the store/api.py